### ISSUE TITLE
Feature: Cable Module for Interface and Circuit connection

### ISSUE THREAD

**Issue by ghost** [netbox-community/ansible_modules#91]
##### ISSUE TYPE
 - Module creation help

##### SUMMARY
Cable module for interface and circuit connections with cable

**Comment by FragmentedPacket** [comment]
@kyungl9 Have you started any work on this? Unclear as to what you need help with?

**Comment by kyungl9** [comment]
Hi I am new to Netbox and I was wondering if you guys have an automation module for cable section under dcim, so that we don't have to manually terminate cables and assgn it to interfaces.

**Comment by FragmentedPacket** [comment]
We currently do not. The supported modules can be found within the README.md or within the `plugins/modules` folder.

We can leave this open and see if anyone else is interested in implementing the module, but currently the priority for me lies in getting the inventory module the requested features and update the testing at the moment.

You or someone else are free to starting working on a module for the Cable endpoint and I'd be more than happy to answer any questions.

**Comment by TawR1024** [comment]
We need this module too. So i'll start implementing it. 
Soon, i'll attach new PR.

**Comment by TerraTalpi** [comment]
I'd need this module too. @TawR1024 how is your status for Pull Request #116 ? It seems that you have stopped work on 26th Feb, a month ago. Are you planning to implement this or should I take over and (try) to implement this based on your work?

**Comment by TawR1024** [comment]
Hey, sory for a log answer. 
unfortunately i haven`t enough time  to work with this module.

I`ll try to end it in near future

**Comment by javeedf** [comment]
@TawR1024 @TerraTalpi any update on this feature?

**Comment by TerraTalpi** [comment]
Hello @javeedf, I did not work on the issue. I have solved it differently by getting the cable endpoint IDs form the ansible modules and then issuing an URI API-call from ansible to connect the two endpoints with a cable. Yeah, this is a dirty hack, but I had to get it working fast. There is currently no plan to implement this from my side because of lack of time in the company and I have to admit, that python is not my strength...
